Dockerfile.assisted-service-onprem

```diff
@@ -31,7 +38,11 @@ ENV COREOS_IMAGE=$WORK_DIR/livecd.iso
 # install config
 # [TODO] - change this line to use openshift-installer from the release, once we are ready
 COPY --from=openshift-installer /root/installer/openshift-install $WORK_DIR
-COPY --from=config-gen $WORK_DIR/process-ignition-manifests-and-kubeconfig.py $WORK_DIR
+COPY --from=config-gen $WORK_DIR/render_files.py $WORK_DIR
```

Comment:
why not to copy all dir?

Reply:
@sacharya Maybe copy *.py?  (Though it doesn't really matter, this will be in Go soon enough)

Reply:
Done.